interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
@kurtwheeler and @mhuyck : I did experience some issues when downloading things into node_modules npm (this is what happened yesterday), since sometimes those servers may be out. However, I agree with @mhuyck that we should keep the `node_modules` folder inside the container itself, as one of its layers, so that this install & build process is reproducible by anyone (both when we deploy the interface files and when somebody else wants to use the container to build the interface reproducibly.)